src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -643,6 +654,21 @@ protected boolean clickImp(String place) {
         return result;
     }
 
+    protected boolean doubleClickImp(String place) {
+        boolean result = false;
+        try {
+            WebElement element = getElementToClick(place);
+            result = doubleClickElement(element);
+        } catch (WebDriverException e) {
+            // if other element hides the element (in Chrome) an exception is thrown
+            String msg = e.getMessage();
+            if (msg == null || !msg.contains("Other element would receive the double click")) {
```

Comment:
Did you see the 'Other element would receive the double click' message in your tests being thrown by Chrome (or did you just copy `clickImpl()` and changed the text?